export/utils.go

```diff
@@ -0,0 +1,178 @@
+package export
+
+import (
+	"encoding/json"
+
+	"golang.org/x/xerrors"
+	corev1 "k8s.io/api/core/v1"
+	schedulingv1 "k8s.io/api/scheduling/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	v1 "k8s.io/client-go/applyconfigurations/core/v1"
+	schedulingcfgv1 "k8s.io/client-go/applyconfigurations/scheduling/v1"
+	cfgstoragev1 "k8s.io/client-go/applyconfigurations/storage/v1"
+)
+
+func ConvertToResourcesForImportFromResourcesForExport(expRes *ResourcesForExport) (*ResourcesForImport, error) {
+	pods, err := convertPodListToApplyConfigurationList(expRes.Pods)
+	if err != nil {
+		return nil, xerrors.Errorf("call convertPodListToApplyConfigurationList: %w", err)
+	}
+	nodes, err := convertNodeListToApplyConfigurationList(expRes.Nodes)
+	if err != nil {
+		return nil, xerrors.Errorf("call convertNodeListToApplyConfigurationList: %w", err)
+	}
+	pvs, err := convertPvListToApplyConfigurationList(expRes.Pvs)
+	if err != nil {
+		return nil, xerrors.Errorf("call convertPvListToApplyConfigurationList: %w", err)
+	}
+	pvcs, err := convertPvcListToApplyConfigurationList(expRes.Pvcs)
+	if err != nil {
+		return nil, xerrors.Errorf("call convertPvcListToApplyConfigurationList: %w", err)
+	}
+	scs, err := convertStorageClassesListToApplyConfigurationList(expRes.StorageClasses)
+	if err != nil {
+		return nil, xerrors.Errorf("call convertStorageClassesListToApplyConfigurationList: %w", err)
+	}
+	pcs, err := convertPriorityClassesListToApplyConfigurationList(expRes.PriorityClasses)
+	if err != nil {
+		return nil, xerrors.Errorf("call convertPriorityClassesListToApplyConfigurationList: %w", err)
+	}
+	return &ResourcesForImport{
+		Pods:            pods,
+		Nodes:           nodes,
+		Pvs:             pvs,
+		Pvcs:            pvcs,
+		StorageClasses:  scs,
+		PriorityClasses: pcs,
+		// existingClusterExportService can't export the SchedulerConfig.
+		SchedulerConfig: nil,
```

Comment:
Now, this `ConvertToResourcesForImportFromResourcesForExport` is utility func. So, set `expRes.SchedulerConfig` here.
In replicateexistingcluster service, `ResourcesForExport.SchedulerConfig` from existing cluster should be already nil. So, just changing here as the above will work well.